projects/plugins/boost/app/assets/src/js/features/lcp/status/error-details.tsx

```diff
@@ -6,6 +6,29 @@ import { useLcpState } from '../lib/stores/lcp-state';
 import styles from './error-details.module.scss';
 
 export const ErrorDetails = () => {
+	const dictionary = {
+		unknown: __(
+			'Something went wrong while optimizing this page. Please try again later, or contact support if the issue persists.',
```

Comment:
I really hope this doesn't end up being "failed to generate critical css" for LCP 😂